### DISCUSSION TITLE
Code review policy - number of required approvals.

### DISCUSSION THREAD

**Discussion by oleg68** [GrandOrgue/grandorgue#1233] [Developers]
Currently our policy requires approving a PR by two rewiing people.

Because we had three developers with change permissions, this policy was adequate: one person pushes a PR, two others approve it.

Especially I have many thanks to @larspalo for a lot of his ideas and comments. Several times I rolled my changes back after his reviews.

Unfortunally Lars dissappeared some time ago, so PRs are waiting for his review for a long time. It causes the GrandOrgue users can not get new releases as quickly as they would like.

I hope Lars will return in the future. But now we have only two developers: @oleg68 and @rousseldenis , so we can not continue with the old approving policy. Would we temporary change it to **one** required approval?

**Comment by rousseldenis** [comment]
@oleg68 Yes we can, but last activity is 15 days ago. If he was on holidays, that can explain this lack of reaction. So, I would not say he disappeared :smiley:

**Reply by oleg68** [reply]
I suggest to switch the policy now to 1 aproval required and to merge all PRs with your approvals now without waiting for @larspalo and to issue 3.9.0.

We could return to the old reviewing policy after his returning.

@rousseldenis what is your opinion?

**Comment by kerkovits** [comment]
I think that it would be respectful to Lars if you wait for one more week. Lars had very good and remarkable comments, and even some serious illness could explain his silence. As a frequent GO user, I would rather wait for a new release for a little time (by the way, if I needed some bugfix, I could use your intermediate builds) than to possibly offend a core developer with our impatience.

**Reply by jason-barba** [reply]
Lars is active at Bureå församling (congregation). If the concern about Lars' status is increasing, a contact to Bureå congregation might be called upon.
https://www.burekyrkan.com/user/1 (in Swedish)

**Comment by larspalo** [comment]
I'm ok! Been too busy with job and family matters to even check my personal e-mail... Sorry, will try to keep up better in the future!

**Reply by jason-barba** [reply]
Good to hear! Take it easy.

**Comment by oleg68** [comment]
@larspalo thank you, @larspalo. I'm very glad to see you here again.

**Reply by larspalo** [reply]
@oleg68 Thanks!
